writing-visualising/dashboards_rshiny.qmd

```diff
@@ -298,11 +298,13 @@ Note that wherever possible dashboard development updates should be done separat
 
 --- 
 
-## Detailed walkthrough of a data update
+### Detailed walkthrough of a data update
+
+--- 
 
 The following provides a more detailed description of how you would do the above steps technically in R-Studio and GitHub if you need a refresher.
 
-### (At least 1 day) Prior to publication
+#### (At least 1 day) Prior to publication
```

Comment:
All these level 4 headings in this section should also have --- 
I can see they didn't have that before either, but for consistency with the rest of the site, we should add them in!